### ISSUE TITLE
Chunk data loading is blocking event loop

### ISSUE THREAD

**Issue by dImrich** [iceiix/stevenarella#679]
I don't understand why is everything except ChunkBuilder and few other things done in main thread. ChunkBuilder's work only takes ~1ms to complete and is done on different thread. On my low end cpu (i5-2400) method load_chunk19_to_117 is taking 4-6ms to complete and in some weird cases even 30ms which is eating the whole frame budget when chunk packets arrive. Is it possible to load and process chunk data on different thread? I can't see why it wouldn't be possible. Stevenarella in this state is just unusable.

**Comment by iceiix** [comment]
Yes I believe this change should be possible. I'm not sure why it was implemented this way with only ChunkBuilder on a  different thread instead of chunk parsing, haven't changed the structure of the chunk loading code since it was imported from Steven.
